### PR TITLE
update winston3

### PR DESCRIPTION
**Pull request by idohalevi**
@AlonMiz 
As we discuss -- dropping splat and giving the user full control on the format. Might break some stuff so I bumped the version

### CODE REVIEW

File: lib/winston-logzio.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 const Transport = require('winston-transport');
 const stringifySafe = require('json-stringify-safe');
 const logzioNodejs = require('logzio-nodejs');
+const {
+  LEVEL,
+  MESSAGE,
+} = require('triple-beam');
```
Comment (AlonMiz) [inline lib/winston-logzio.js]: y do we need this? seems like a less supported library

File: test/winston-logzio.js [hunk 2]
```diff
@@ -24,93 +30,127 @@ describe('winston-logzio', () => {
...
+      assert(loggedObject[MESSAGE] === loggedObject.message);
 
       done();
     });
   });
 
-  describe('send json as log message without error', () => {
-    it('builds the log object with just a message object', (done) => {
+  describe('send json as log message', () => {
+    it('bbuilds the log object using json formater', (done) => {
```
Comment (AlonMiz) [inline test/winston-logzio.js]:
```suggestion
    it('builds the log object using json formater', (done) => {
```

File: lib/winston-logzio.js (hunk 1 above)
Comment (idohalevi) [reply lib/winston-logzio.js]: @AlonMiz 
That's how winston uses it. For example https://github.com/winstonjs/logform/blob/master/logstash.js

Comment (idohalevi) [reply lib/winston-logzio.js]: Also, in case of message field missing in `info` we need to take the splat(message) from `info`

**Pull request merged**